libs/cri-storing-service/src/main/java/uk/gov/di/ipv/core/library/cristoringservice/CriStoringService.java

```diff
@@ -121,6 +122,9 @@ public void storeVcs(
 
             verifiableCredentialService.persistUserCredentials(vc);
             ipvSessionItem.addVcReceivedThisSession(vc);
```

Comment:
(You'll also need to skip these steps for the TICF CRI, but assuming that'll be in a later PR...)

Reply:
yeah this change to just start storing ticfVC to new column for session.